### PR TITLE
Design Components

### PR DESCRIPTION
**Pull request by didierganthier**
## Checklist
- [x] I have read this repository's contribution guidelines.
- [x] My pull request has a descriptive title (not a vague title like Update index.MD)
- [x] My pull request targets the dev branch of this [project](https://github.com/didierganthier/javascript-group-capstone).
- [x] I added relevant documentation for all the files I changed

This commit adds the linters file to handle local linters checks and GitHub Actions automation

 ## What changes have been made?
- [x] Design a `Display.js` component to display the user's pick
- [x] Design a `Button.js` component to be called in a `Keypad.js` component
- [x] Design a `Keypad.js` component to show the buttons
- [x] Design a `Calculator.js` component to call both `Display.js` and `Keypad.js` as it's children
- [x] Pass `Calculator.js` as children of `App.js` 

The author of these changes is the owner of the repo @didierganthier

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (Ismailco) [inline src/App.js]: - [ ] The App.js component should be class-based, kindly consider implementing this so the component is class-based

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @didierganthier, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
- [ ] Kindly consider removing all boilerplate that got created by the CRA (e.g App.test.js, reportWebVitals.js, setupTests.js...)
- [ ] 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (didierganthier) [reply src/App.js]: Thanks @Ismailco I'm fixing this 😎

**Review by alexander16108 — APPROVED** [review]
Hi @didierganthier , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**